### PR TITLE
Corrected poor design of UI. For issue  #541 

### PR DESCRIPTION
**Pull request by harsh-kukreja**
- Overlay mobile view aligned. 
- Changed the overlay text color to white. 
- Fixed pixelated images
For issue  #541

### CODE REVIEW

**Comment by harsh-kukreja** [comment]
**Screenshot of the result**
Capture

**Comment by harsh-kukreja** [comment]
Hey @mariobehling the issue of #474 got pushed here iteself is it fine??

File: css/custom.css [hunk 1]
```diff
@@ -175,15 +175,15 @@ nav{
 }
  .has-submenu{
   position: relative;
-} 
+}
```
Comment (mariobehling) [inline css/custom.css:178]: What is this change?

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please don't change the images to be so big. 
Please provide a test deployment link.

File: index.html [hunk 2]
```diff
@@ -1,1667 +1,2064 @@
...
+    <link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/custom-blog.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/bootstrap-theme.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/text-overlay.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+    <script>
+        (function (i, s, o, g, r, a, m) {
```
Comment (kushthedude) [inline index.html:37]: These are a lot of unwanted change. 
Please make changes relevant to your issue.

**Comment by harsh-kukreja** [comment]
Hey @kushthedude and @mariobehling sorry for the trouble but i mistakenly pushed two issue solution into the same PR what can i do to rectify this can you please help me or do you want me to create a new PR for both issue and closing the current PR's 
The issue numbers i am talking is #474  and #541

**Comment by harsh-kukreja** [comment]
> Please don't change the images to be so big.
> Please provide a test deployment link.

I just picked up the images from play store itself and added a class of img-responsive of bootstrap

**Review by mariobehling — CHANGES_REQUESTED** [review]
It is difficult to review all the changes as you somehow moved or changed e.g. 3000+ lines of code in the index file. Seems like you used an automatic tool to clean up something maybe? Please only make the changes related to your issue. Thanks.

**Comment by mariobehling** [comment]
> Please provide a test deployment link.

**Comment by harsh-kukreja** [comment]
Hey @mariobehling here is the development link for this PR [Link](https://harsh-kukreja.github.io/harsh-fossasia.org/)

File: index.html [hunk 3]
```diff
@@ -1,1667 +1,2064 @@
...
-						</div>
-					</div>
-
-					<div class="row">
-
-						<div class="col-sm-6 col-md-4">
-							<div class="topic">
-								<i class="pe-7s-loop icon-large"></i>
-								<h3>What We Do<br></h3>
-								<p class="lead">We bring amazing people together to create, develop, design and make things with Open T
```
Comment (harsh-kukreja) [inline index.html:614]: Hey @kushthedude and @mariobehling i changed the HTML for this div of row I didn't use any tool i just removed the button back to top issue's code sorry for the trouble

File: css/custom.css (hunk 1 above)
Comment (harsh-kukreja) [reply css/custom.css:178]: I have marked the changes where i did sorry for the trouble

File: css/text-overlay.css [hunk 4]
```diff
@@ -1,17 +1,7 @@
-.container {
-  position: relative;
-}
 
 .text-block {
-  position: absolute;
-  bottom: 80px;
-  right: 80px;
-  left: 50%;
+  position: relative;
```
Comment (harsh-kukreja) [inline css/text-overlay.css:3]: And added 2 lines of css LineNumber 3,4

**Pull request closed without merging**